nosql_test.go

```diff
@@ -310,12 +309,42 @@ func TestMySQL(t *testing.T) {
 	run(t, db)
 }
 
+func TestMongoDB(t *testing.T) {
+	var (
+		uname = "user"
+		pwd   = "password"
+		addr  = "localhost:27017"
+		// path   = "/tmp/mysql.sock"
+		replicaSet = "dbrs"
+		testDB     = "test"
+	)
+
+	isCITest := os.Getenv("CI")
+	if isCITest == "" {
+		fmt.Printf("Not running MongoDB integration tests\n")
+		return
+	}
+
```

Comment:
Might be better to actually _do_ run a test in CI. Maybe can be done as a next step while improving our `nosql` package, @azazeal? Was thinking of maybe using `testcontainers`, https://github.com/benweissmann/memongo or https://github.com/ONSdigital/dp-mongodb-in-memory.

Reply:
For now, instead of `fmt.Printf` & `return`, please use `t.Skip` with an appropriate message that does not start with a capital letter.